AdminCore.Services.Tests/MockDatabase.cs

```diff
@@ -17,6 +17,7 @@ public class MockDatabase
     public List<EventStatus> EventStatusRepository;
     public List<EventType> EventTypeRepository;
     public List<Team> TeamRepository;
+    public List<MandatoryEvent> MandatoryEventRepository;
```

Comment:
These changes can be ignored for now as this class will be deleted when DashboardService tests are updated